templates/cli/lib/questions.js.twig

```diff
@@ -152,7 +153,9 @@ const questionsInitProject = [
         message: "Choose your organization",
         choices: async () => {
             let client = await sdkForConsole(true);
-            const { teams } = await paginate(teamsList, { parseOutput: false, sdk: client }, 100, 'teams');
+            const { teams } = /.*appwrite\.io/.test(client.endpoint)
```

Comment:
hmm can we use a simpler way rather than using regex here? Maybe a string endswith or so? Its more readable that way